plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibraryContainerResolverJob.java

```diff
@@ -147,29 +148,29 @@ private LibraryClasspathContainer resolveLibraryFiles(ILibraryRepositoryService
     return container;
   }
 
-  private int getTotalwork(IClasspathEntry[] rawClasspath) {
+  private static int getTotalwork(IClasspathEntry[] rawClasspath) {
```

Comment:
This seems like overkill: just make the total work the number of libraries, and if it's not a library classpath entry, then you move on.